### ISSUE TITLE
tests.foreman.ui.test_activationkey.ActivationKey.test_associate_product_1

### ISSUE THREAD

**Issue by sghai** [SatelliteQE/robottelo#2662]
Stacktrace
```
  File "/usr/lib64/python2.7/unittest/case.py", line 369, in run
    testMethod()
  File "/home/jenkins/workspace/satellite6-automation-downstream-rhel71-ui/tests/foreman/ui/test_activationkey.py", line 1192, in test_associate_product_1
    repo_id = self.enable_sync_redhat_repo(rh_repo, org_id=org_id)
  File "/home/jenkins/workspace/satellite6-automation-downstream-rhel71-ui/tests/foreman/ui/test_activationkey.py", line 87, in enable_sync_redhat_repo
    releasever=rh_repo['releasever'],
  File "robottelo/api/utils.py", line 28, in enable_rhrepo_and_fetchid
    query={'organization_id': org_id})[0].id
'list index out of range\n-------------------- >> begin captured logging << --------------------\nnailgun.client: 
```

**Comment by sghai** [comment]
Looks like this UI test is failing due to recent fixes in nailgun.

**Comment by sghai** [comment]
Another test failing with same error:
`tests.foreman.ui.test_activationkey.ActivationKey.test_positive_update_activation_key_with_rh_product_1`

**Comment by sghai** [comment]
two more:
`tests.foreman.ui.test_contentviews.TestContentViewsUI.test_cv_promote_rh_1`
`tests.foreman.ui.test_contentviews.TestContentViewsUI.test_cv_publish_rh_1`

**Comment by oshtaier** [comment]
Just a guess.
@sthirugn resolved 'releasever' issue in his latest PR: #2661 'Bug fix for passing None to Release Version'. Maybe, we can apply something similar for current issues.
Need to investigate more
